instana/instrumentation/celery/hooks.py

```diff
@@ -55,8 +56,11 @@ def task_prerun(*args, **kwargs):
             headers = task.request.get('headers', {})
             if headers is not None:
                 ctx = tracer.extract(opentracing.Format.HTTP_HEADERS, headers)
+                active_tracer = tracer
+            else:
+                active_tracer, parent_span = get_active_tracer()
 
-            scope = tracer.start_active_span("celery-worker", child_of=ctx)
+            scope = active_tracer.start_active_span("celery-worker", child_of=ctx)
```

Comment:
I guess this will throw an exception if there is no active tracer and `headers` are `None`, right?